packages/logger/test/unit/basic-specs.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable no-console */
 import { Log } from '../../lib/log';
+import { unleakString } from '../../lib/utils';
```

Comment:
I was not quite sure about keeping this method in support
The reason for that is that support has dependencies on logger
And we need to unleakString to be used in logger, which creates a circular dependency.
I have copied unleakString to the logger because of that reason, although I can also observe this method is not used anywhere else except of logger.
Perhaps, it makes sense to remove it from support for good since we anyway bump the major version of it

Reply:
cc @KazuCocoa

Reply:
This import pulls from the unleakString exported by the logger package. The unleakString in the support package will be removed as part of #20221.

Reply:
why do we test it here then? it would make more sense to add tests in the logger module then

Reply:
This is the logger module :)

Reply:
ah, yes. Sorry for the confusion